doc/docker/jenkinsnode/Dockerfile

```diff
@@ -0,0 +1,95 @@
+FROM debian:stretch
+
+ARG SWIGVERSION="3.0.11"
+
+# install dependencies
+RUN apt-get update && apt-get -y upgrade
+RUN apt-get -y install \
+  apt-utils \
+  curl \
+  build-essential \
+  clang \
+  autotools-dev \
+  automake \
+  cmake \
+  pkg-config \
+  doxygen \
+  graphviz \
+  bison \
+  ruby-dev \
+  python-dev \
+  python3-dev \
+  libpython3-dev \
+  liblua5.3-dev \
+  tclcl-dev \
```

Comment:
Not needed! The TCL plugin does not have dependences.

Reply:
copied from the a7, i'll try removing them to see if it still works afterwards